Classes/ToxWrapper/OCTTox.h

```diff
@@ -389,6 +389,36 @@ extern const NSUInteger kOCTToxPublicKeyLength;
  */
 - (int)fileSendRequestWithFriendNumber:(int32_t)friendNumber fileName:(NSString *)fileName fileSize:(uint64_t)fileSize;
 
+/**
+ * TODO : Add info about method
+ */
+- (int)fileSendControlWithFriendNumber:(int32_t)friendNumber sendReceive:(uint8_t)sendReceive fileNumber:(uint8_t)fileNumber controlType:(uint8_t)controlType data:(const uint8_t *)data length:(uint16_t)length;
+
+/**
+ * TODO : Add info about method
+ */
+- (int)fileSendDataWithFriendNumber:(int32_t)friendNumber fileNumber:(uint8_t)fileNumber data:(const uint8_t *)data length:(uint16_t)length;
```

Comment:
- return `BOOL`
- `data` and `length` should be replaced with one `NSData *` parameter.